src/components/lib/VDropdownSecondary/VDropdownSecondary.vue

```diff
@@ -102,7 +104,14 @@ watch(
         </span>
       </span>
       <span v-else class="placeholder">{{ placeholder }}</span>
-      <div class="arrow"></div>
+      <img
+        :src="arrowIcon"
```

Comment:
Same as this https://github.com/arcana-network/developer-dashboard/pull/419/files#r1658129274